src/app/config/config.model.ts

```diff
@@ -78,6 +80,16 @@ interface TPLinkSmartPlugPlugin extends Plugin {
   smartPlugIP: string;
 }
 
+interface TasmotaPlugin extends Plugin {
+  ip: string;
+  idx: number;
+}
+
+interface TasmotaMqttPlugin extends Plugin {
+  topic: string;
+  relayN: number;
```

Comment:
I think a more describing attribute name would also be a better fit here. If its the relay number `relayNumber` might be a good option.